toolkit/imageconfigs/marketplace-gen1-fips.json

```diff
@@ -67,6 +67,9 @@
             "PostInstallScripts": [
                 {
                     "Path": "additionalconfigs/configure-image.sh"
+                },
+                {
+                    "Path": "postinstallscripts/remove-tdnf-cache.sh"
```

Comment:
The change looks fine but I'd consider these two modifications to either:
- add the contents of the new script into `configure-image.sh` or
- re-name `configure-image.sh` to something more precise like `configure-systemd-networkd.sh`.

Currently, I'm leaning more towards the second option, so the configuration scripts are more granular and their responsibility is clearer.

Reply:
Added the renaming